test/image/mocks/layout_image.json

```diff
@@ -4,17 +4,22 @@
       "x": [1,2,3],
       "y": [1,2,3]
     }, {
-      "x": [1,2,3],
-      "y": [1,2,3],
+      "x": ["2001-01-01","2002-01-01","2003-01-01"],
```

Comment:
I see you're catching on to the _reusing existing mocks_ philosophy. Thanks! Our CI test time will thank you too